src/calendar/view/EventPreviewView.js

```diff
@@ -78,13 +79,17 @@ export class EventPreviewView implements MComponent<Attrs> {
 	}
 
 	_renderAttendee(attendee: CalendarEventAttendee): Children {
+		let attendeeField = attendee.address.address
+		if(downcast(attendee.address)._errors){
```

Comment:
We should make some utility function to check for errors.

Reply:
Do you mean just for the EncryptedMailAddressType or more generic?

Reply:
For any entity, yes

Reply:
Like this?
`export function hasError<T, K>(instance: T, key: ?K): boolean {
	return !instance || (!!instance._errors && (!key || !!instance._errors.key))
}`

Reply:
Yeah except if you don't restrict `T` it's effectively `any`. Maybe you could use `SomeEntity` if it works.

Reply:
Yeah then we need `downcast()`. I pushed it.